### ISSUE TITLE
Spring Proxy

### ISSUE THREAD

**Issue by qnradkaft** [vaadin/spring#171]
I have this kind of error after login to vaadin with spring integration application:

Widgetset 'towidgetset' does not contain implementation for com.sun.proxy.$Proxy143. Check its component connector's @Connect mapping, widgetsets GWT module description file and re-compile your widgetset.

I have found that this error concern spring vaadin - integration. Error is not showing all the time but it is very random. Sometimes app is running without this error.
Anybody know how to solve this? Any idea?

**Comment by frezelth** [comment]
Hi,
Same issue here, did you find the cause of the problem?

**Comment by qnradkaft** [comment]
No, We do javascript workaround with click differrent element on web page,
then this error diseppear in our app.

2017-03-07 11:48 GMT+01:00 Thomas Frezel <notifications@github.com>:
